nemo/collections/common/data/lhotse/cutset.py

```diff
@@ -394,14 +395,25 @@ def read_nemo_manifest(config, is_tarred: bool) -> CutSet:
     metadata_only = config.metadata_only
     if isinstance(config.manifest_filepath, (str, Path)):
         logging.info(f"Initializing Lhotse CutSet from a single NeMo manifest (tarred): '{config.manifest_filepath}'")
+
         if is_tarred and not metadata_only:
-            cuts = CutSet(
-                LazyNeMoTarredIterator(
-                    config.manifest_filepath,
-                    tar_paths=config.tarred_audio_filepaths,
-                    **common_kwargs,
+            if config.tarred_random_access:
```

Comment:
After you merge `main` you will have conflicts in this file due to a new option `force_finite` that applies `.repeat()` only when set to `False`. When resolving these conflicts, ensure that when `tarred_random_access` is `True`, `force_finite` is also set and pass `config.tarred_random_access` directly to `LazyNeMoTarredIterator` (without adding a new `if/else` as done here).